packages/apple-pay-integration/e2e/ApplePayTestMockResponse.ts

```diff
@@ -527,3 +527,452 @@ export const consignmentsAndBilling = `{
     ],
     "updatedTime": "2022-08-10T06:12:15+00:00"
 }`;
+
+export const checkoutSettingsBodyMock = `{
```

Comment:
What makes this mock work for your e2e tests?
I'm trying to figure out what is missing in the existing HAR files.

Reply:
I mocked it for modification if necessary. I was testing with integration store and paid attention on this feature `'CHECKOUT-6891.update_incomplete_order_wording_on_order_confirmation_page'`. This feature must be false to work correctly. But I think I can turn it off for the store integration, and if that's the case, we don't need this mock anymore. I'll have to look into it.

Reply:
@bc-peng removed this mock and updated apple pay HAR files